webpack.renderer.config.js

```diff
@@ -117,11 +117,7 @@ const rendererConfig = {
     new webpack.DefinePlugin({
       "process.env.RELEASE": `"${commitHash}"`
     }),
-    new webpack.DefinePlugin({
-      "process.env": {
-        NODE_ENV: JSON.stringify(process.env.NODE_ENV)
-      }
-    }),
+    new webpack.EnvironmentPlugin([`NODE_ENV`, `RELEASE`, `STARGATE`, `RPC`, `RELEASE`, `FAUCET`]),
```

Comment:
```suggestion
    new webpack.EnvironmentPlugin([`NODE_ENV`, `STARGATE`, `RPC`, `RELEASE`, `FAUCET`]),
```

Reply:
Thanks! I think I will fall back on the dear old DefinePlugin, MacOS and linux behave slightly differently (or can be just different types of shells) with this one, it seems to just forward